import java.util.Scanner;

public class Q3_demo 
{
    public static void main (String[] args)
    {
        try{
            try{
              int z=0;
        int b=4/i;
        System.out.println("Not executed");  
            }
        catch(ArithmeticException e)
        {
            System.out.println("Division by zero error");
        }
    catch(Exception e)
        {
            System.out.println("");
        }
    }
        catch(ArithmeticException e)
        {
            System.out.println("Division by zero error");
        }
        System.out.println("Outside catch block");

    
    }
}